include/swift/AST/KnownSDKDecls.def

```diff
@@ -0,0 +1,26 @@
+//===--- KnownSDKDecls.def - Compiler decl metaprogramming     --*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines macros used for macro-metaprogramming with compiler-known
+// declarations, in modules other than the standard library.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef KNOWN_SDK_FUNC_DECL
+#  define KNOWN_SDK_FUNC_DECL(Module, Name, Id)
+#endif
+
+//KNOWN_SDK_FUNC_DECL(Distributed, MissingDistributedActorTransport, "_missingDistributedActorTransport")
+KNOWN_SDK_FUNC_DECL(Distributed, IsRemoteDistributedActor, "__isRemoteActor")
```

Comment:
I'll rename this function soon I think, but the important bit is that it's just in the distributed module